dds_cli/__main__.py

```diff
@@ -11,40 +11,40 @@
 import os
 import pathlib
 import sys
-from logging.config import dictConfig
 
 # Installed
 import click
 import click_pathlib
-import questionary
+import requests
 import rich
-import rich.console
 import rich.logging
+import rich.progress
 import rich.prompt
-from rich import pretty
-from rich.progress import Progress, BarColumn
+import simplejson
+import questionary
 
 # Own modules
 import dds_cli
+import dds_cli.admin
```

Comment:
This causes an error on the CLI, I assume its something you'll add in the next PR. But probably good to delete it here so it's not breaking the cli